docs/api/geo_datasets.csv

```diff
@@ -18,6 +18,7 @@ Dataset,Type,Source,Size (px),Resolution (m)
 `Landsat`_,Imagery,Landsat,"8,900x8,900",30
 `NAIP`_,Imagery,Aerial,"6,100x7,600",1
 `NLCD`_,Masks,Landsat,-,30
-`NCCM`_,Masks,Sentinel-2,-,10 
+`NCCM`_,Masks,Sentinel-2,-,10
 `Open Buildings`_,Geometries,"Maxar, CNES/Airbus",-,-
+`PRISMA`_,Imagery,PRISMA,512x512,5--30
```

Comment:
The files I was given are 512x512, need to see if that's the size of all files or just this subset.